openquakeplatform/openquakeplatform/irv/static/irv/js/irv_viewer.js

```diff
@@ -1258,27 +1257,21 @@ function thematicMapCreation() {
     $('#absoluteSpinner').hide();
 }
 
-function watchForPdSelection() {
+function whenProjDefSelected() {
+    $('#pdSelection').prop("disabled", true);
```

Comment:
Done. I'm keeping it also here, because the updating process triggered by the selection might take time, and we should not enable selecting a different project definition until such operation has been completed.